tests/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+include_directories(.)
```

Comment:
Is this intentional?

Reply:
Yes. It's backported from devel and allows for includes relative to `tests/`, e.g. `#include "Mocks/Servers.h"`, even if you aren't in the `Mocks` directory.